src/lib/util/editor/quill.ts

```diff
@@ -7,7 +7,9 @@ import { trpc } from '$lib/trpc/client';
 import type { Page } from '@sveltejs/kit';
 import type { DeltaProperties } from '$lib/shared/delta';
 import { writable } from 'svelte/store';
-import type { IllustrationObject } from './quill.illustration';
```

Comment:
Please remove the util/editor/quill.illustration file since it's not longer being used.

Reply:
Fixed